content/en/docs/release-oversight/disruption-tests.md

```diff
@@ -0,0 +1,213 @@
+---
+title: "Disruption Tests"
+description: An look at how disruption tests are architected and implemented.
+---
+
+## Summary
+
+Disruption tests are a set of tests that measure an endpoints resilience or an allowed alert duration. As the `e2e tests` are run the results are matched against historical data to determine if the disruption intervals are within expectations.
```

Comment:
s/endpont/endpoint's/

we are measuring disruption which is defined as the amount time an endpoint is not responding to requests (e.g., during an upgrade).  The disruption testing is done independent of alerts checking.

Reply:
I just saw your link on origin/pkg/synthetictests/allowedalerts/types.go and wonder if this is part of disruption testing -- I don't know the answer to this.

Reply:
Good point, the intent here was to try and cover both, from my understanding the way that the disruption and alert numbers are gathered is the same. Even the matcher is the same, the only thing that is different is the historical data set which we check against. Maybe we say this is documentation on how we determine allowed tolerations?

Reply:
I think describing both is a good idea but I think since they are quite similar, I recommend just mentioning the difference as you just explained.